testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergSparkCompatibility.java

```diff
@@ -119,8 +139,8 @@ public void testSparkReadingTrinoData()
                         ", _boolean BOOLEAN" +
                         //", _timestamp TIMESTAMP" +
                         ", _date DATE" +
-                        ") WITH (format = 'ORC')";
-        onTrino().executeQuery(format(trinoTableDefinition, trinoTableName));
+                        ") WITH (format = '%s')";
+        onTrino().executeQuery(format(trinoTableDefinition, trinoTableName, storageFormat.name()));
```

Comment:
Since this is an enum just for tests, we can ensure that `toString()` is the same as `name()` and simplify this code. This is one of those places where it's easy to get wrong (use string instead of name method) and you wouldn't notice since they are actually the same, so it's better to enforce they are the same.

Reply:
> Since this is an enum just for tests, we can ensure that toString() is the same as name() and simplify this code. 

I intentionally used `name`. As in `io.trino.tests.product.hive.TestHiveStorageFormats.StorageFormat` we may need to replace enum with a complex object and then using `name` here helps us find places that require an update.
for example, i don't want to rely on spark's logic when it's given an unrecognized `write.format.default`. (it should fail, i guess, but will it?)
i don't need the `.name()` for Trino's `format`, but using everywhere for consistency

Reply:
I’m not convinced by this argument of “may need to replace”. There are exactly three values. They either work or they don’t. If we ever did need to change it, finding usages of the class would be fine.

Reply:
if you're planning to work on this tests a lot in the future, i will give you way. but in case you don't have such plans, i would prefer to use the approach that's easier to work with from my perspective